documentation/docs/node_api_core.md

```diff
@@ -0,0 +1,509 @@
+# `Client::get_health`
```

Comment:
Why not just link/include the Rust docs for this kind of stuff? Don't really see added value in this

Reply:
Yeah I was kind of thinking the same which is why I didn't actually fill the request/response yet. I'd still like to have this page to give the example commands, so I'll probably end up reducing the content a lot and link to the rust docs. I'll rework this doc page when I'm done with all the example if you're fine with it.